### DISCUSSION TITLE
New Home Assistant Add-On configuration for newbies

### DISCUSSION THREAD

**Discussion by jazzmonger** [toverainc/willow#355] [General]
https://community.home-assistant.io/t/espressif-box-devices-willow-multilingual-voice-stt-tts-home-assistant-add-on-how-to-configure/664916

**Comment by senna1992** [comment]
Another hint, that hopefully saves you some hassle with the first test runs :

It is very command sensitive. 
Alexa turn on the coffeemaker <-- works
Alexa schalte die Kaffeemaschine ein <-- geht

Alexa schalte das Licht in der Küche ein <-- geht 
Alexa turn on the kitchen light <-- doesnt work
Alexa schalte das Licht in der Küche an <-- geht nicht 
Aelxa turn on the light in the kitchen <-- works

Is there any Guide on how to use willow autocorrect?
I've set it up and added the HA token, but it does seem like i still miss on something!

If anyone can help, i'd highly apprechiate :)
